website/project/metadata/asist-results-registration.json

```diff
@@ -1,67 +1,69 @@
-               "example":"Last version of the testbed used: 1.0.n https://gitlab.asist.aptima.com/XXXX",
-               "required":true
-            },
-            {
-               "qid":"q5",
-               "type":"string",
-               "format":"text",
-               "title":"Dataset used",
-               "description":"Please link to the datasets used in running your the preregistered plans/analyses.",
-               "example":"The following datasets were used in running our preregistered analyses: https://console.cloud.google.com/XXXX",
-               "required":true
-            },
-            {
-               "qid":"q6",
-               "title":"Results registration",
-               "type":"osf-upload",
-               "format":"osf-upload-open",
-               "description":"Please attach the file containing your hypothesis/capability preregistration and results.",
-               "required":true
-            }
-         ]
-      }
-   ]
-}
\ No newline at end of file
+  "atomicSchema": true,
+  "name": "ASSIT Registration Results",
+  "version": 1,
+  "description": null,
+  "blocks": [
+    {
+      "block_type": "page-heading",
+      "display_text": "Results Registration Information"
+    },
+    {
+      "block_type": "question-label",
+      "help_text": "Please provide a brief summary of the results enclosed in your results registration document. This includes results of analyses set forth in the preregistration and results of any post hoc analyses.",
+      "display_text": "Results summary",
+      "example_text": "Hypothesis #1 was confirmed: Navigation skill determines search strategy (p = 0.023). Hypothesis was not confirmed: H2 was not confirmed: Triage is more efficient before anticipated death of high value victims than after (p = .273).  "
+    },
+    {
+      "block_type": "long-text-input",
+      "required": true
+    },
+    {
+      "block_type": "question-label",
+      "help_text": "Please provide a link to the study preregistration this study relies upon.",
+      "display_text": "Study preregistration link"
+    },
+    {
+      "block_type": "short-text-input",
+      "required": true
+    },
+    {
+      "block_type": "question-label",
+      "help_text": "Please provide a link to the preregistration this results registration is following upon.",
+      "display_text": "Preregistration link"
+    },
+    {
+      "block_type": "short-text-input",
+      "required": true
+    },
+    {
+      "block_type": "question-label",
+      "help_text": "Please provide the testbed version used (please do not share the link)",
+      "display_text": "Testbed used",
+      "example_text": "Results were generated using ASIST Testbed version [insert version]. This testbed is currently available only to registered performers on this DARPA program. The testbed will become publicly available by July 2024 at a site to be determined. The testbed versions used ranged from: [insert version] to [insert version]."
```

Comment:
>ASIST

I'm not sure if this is supposed to be a rename, but at least one of these two is wrong.